code/game/machinery/excelsior/redirector.dm

```diff
@@ -0,0 +1,160 @@
+/obj/machinery/excelsior_redirector
+	name = "excelsior ship-navigation redirector"
+	desc = "A arcane mechanism used to hack the bluespace direction control of ships and inevitably lock their course towards excelsior-controlled space. Only works on navigation consoles made before 2502"
+	description_antag = "The final objective of the revolution. Install this ontop of the main navigation console of the ship, located in the bridge and defend it while it does its job"
+	density = TRUE
+	anchored = FALSE
+	icon = 'icons/obj/machines/excelsior/redirector.dmi'
+	icon_state = "redirector_unanchored"
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 0
+	active_power_usage = 0
+	circuit = /obj/item/electronics/circuitboard/excelsior_navigation_cracker
+	var/oldSecurityLevel = null
+	var/timeStarted = null
+	var/timeRebooted = null
+	var/rebootTimer = null
+	var/redirectTimer = null
+	var/antennaBent = FALSE
+
+/obj/machinery/excelsior_redirector/examine(mob/user, distance, infix, suffix)
+	. = ..()
+	if(is_excelsior(user))
+		to_chat(user, SPAN_DANGER("Do not build any walls around this as it will interfere with the mechanism and cause it to instantly fail"))
```

Comment:
```suggestion
		to_chat(user, SPAN_DANGER("Do not build any walls around this as it will interfere with the mechanism and cause it to instantly fail."))
```